client/startup.js

```diff
@@ -47,15 +60,53 @@ const configurePageLoadTracking = () => {
   }, false);
 };
 
-Meteor.startup(() => {
+const analyticsStartup = () => {
+  // TODO: SETTINGS needs to be factored out...maybe passed in somehow
   if (SETTINGS) {
     analytics.initialize(SETTINGS);
```

Comment:
@ccuilla @pauldowman We should add the suggested fix here when `analytics.initialize(SETTINGS)` to resolve the `autorun: false` bug. See comment here https://github.com/okgrow/analytics/issues/142#issuecomment-259808250